### PR TITLE
Nerfs the smoke machine's max range + reagent capacity + decapitalizes name and gives a good description

### PR DESCRIPTION
**Pull request by praisenarsie**
goofball was right.

:cl: i ded
balance: Smoke Machine now only has a max setting of 9, downed from 15.
balance: Smoke Machine can now only hold 300 worth of reagents rather than 1000.
/:cl:

i ded

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by Robustin** [comment]
Show me on this doll where the smoke machine touched you. 

I've yet to see this machine kill anyone except the time I filled it spraytan and cfl3, the machine was disabled in less than 60 seconds and I killed 2 people with 10 minutes of prep, plus I'm nerfing the spraytan stun.

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm [hunk 1]
```diff
@@ -11,9 +11,9 @@
 	var/cooldown = 0
 	var/screen = "home"
 	var/useramount = 30 // Last used amount
-	var/volume = 1000
+	var/volume = 300
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/machinery/smoke_machine.dm]: stealth nerf

**Comment by Iamgoofball** [comment]
@Robustin i like this machine, it is good

it should stay as is tbh

**Comment by Iamgoofball** [comment]
additionally yeah the spraytan crap was a dumb oversight, not robustin's fault

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (praisenarsie) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: 🚶

**Comment by praisenarsie** [comment]
where's my i ded label

Anyway, the smoke machine is dumb and it's a free 1000 reagent grenade with a 15 tile range. If you want, I can roll it into a special traitor item for chemist/CMO or make it only available via cargo. Otherwise, I'm staying with this nerf.

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (JJRcop) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: no put this in the pr description, not tolerated

**Comment by GuyonBroadway** [comment]
Yeah but put the capacity nerf in the changelog ya fookin' ding dong!

**Comment by praisenarsie** [comment]
@GuyonBroadway i REALLY ded

File: code/modules/reagents/chemistry/machinery/smoke_machine.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: ^^^^^^^^^^^^^^^^^^^^^

Comment (praisenarsie) [reply code/modules/reagents/chemistry/machinery/smoke_machine.dm]: 🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶🚶

**Comment by GuyonBroadway** [comment]
Then lets EVERYONE know how ded you are.

**Comment by praisenarsie** [comment]
was too ded to figure that out

**Comment by Bawhoppen** [comment]
>literally capitalized the name of it

Robustin, do you not know our code standards or something?

**Comment by Robustin** [comment]
So you were one of the two people I killed who typed "Robustin im going to nerf your smoke machine" in deadchat?

Fucking awful dude. It's been out for a week and you're the first casualty so you run crying to Github about it and say you want to make it traitor only? Just fuck off.

**Comment by praisenarsie** [comment]
> the smoke machine is dumb and it's a free 1000 reagent grenade with a 15 tile range.

you're should be very glad i was lazy and didn't remove this or roll it into cargo or traitor.

**Comment by JJRcop** [comment]
your pr title is really long, for future reference, you don't have to put all the details in it

**Comment by praisenarsie** [comment]
come on you didn't even add the smoke machine to the bioweapons base

**Comment by Qbopper** [comment]
smoke machine + space cleaner let the crew instantly clean the entire station when I told them an inspector was coming so the inspector couldn't yell at them anymore, remove the smoke machine NOW

for real though this was one of the messiest rounds I had ever seen and it was pristine in like 10 minutes, the smoke machine is a little bit crazy in how low effort it is

**Comment by Iamgoofball** [comment]
Smoke shouldn't be working on tiles.

**Comment by sawrge** [comment]
almost every round I've played has been smoke machine + colourful reagant + dirt + carpet forever it's soooooooo epic

**Comment by jughu** [comment]
Uhh no @robustin that was me

**Comment by jughu** [comment]
Anyways proper fix is again making it to tier up with stock parts then straight up nerfing it both sides happy its not like chemistry can make BIG BOOM grenades at that point anyway

**Comment by Arianya** [comment]
It's a item that bypasses our existing chemistry system, starts prebuilt and can spread 15 tiles, can hold more chemicals then even a large beakers grenade is the most barefaced power creep we've ever seen for chemistry.

Also probably the stock parts thing is a good idea but tbh I'm not unhappy with this going through.

**Comment by Qbopper** [comment]
i've been getting on more often now and I'm floored this got merged as is, the range is ridiculous and it takes nearly no effort to use

the idea is cool but this is not something that should be able to fill the screen with chemicals faster than grenades and without actual effort being put into setup

**Comment by GuyonBroadway** [comment]
Its 10x worse (or better depending on how you use it) on hippie because smoke can leave behind chempiles.

It does mean that if you cover the hallway in healing fruit juices and hearty punch that critical patients headed to medbay tend to live longer.

**Comment by Robustin** [comment]
The only thing flooring here is how we've managed to churn up dozens of people offering theory-crafted input instead of an actual example of why this thing needs its range reduced.

Downsides compared to a smoke grenade:
1) Requires power
2) Can't fit in your pocket, has to be dragged around
3) Anyone can access it and purge its contents
4) It only has a single tank so all of the powerful reagent reactions (explosions) cannot be used 
5) The smoke is transparent and doesn't hinder visility
6) It has to be wrenched down before it can be used, the wrenching effect is twice as long as other anchor-toggled machinery.

@Qbopper 

I took that route with only giving Chemistry a circuit board for it but Bawhoppen fucking threw that board into Tech Storage so I had to insist on putting the machine inside chemistry. 

If the long range is being used to disperse something you find annoying, put on the damn internals you start the game with and purge it, or ask the AI to purge it if you're even lazier still. If its producing something useful, why bitch about it? 

The only combination for this machine that isn't countered by the thing every player gets (internals) at roundstart is lube + CFL3 but now that the smoke isn't even opaque you can easily tell that the smoke is burning and the fire will destroy the machine itself within 60 fucking seconds which is a terrible trade-off for antagonistic activity. Not to mention if you use that combo you're absolutely going to die in your own lubefire.

Yes this was intended to boost chemistry's power. Smoke was an almost unused mechanic because it never worked for the good-guys (nobody is going to stand in your "healing smoke" until they verify what it is, which they can't without medical supplies and they wouldn't need healing smoke if they had medical supplies) and the bad-guys (smoke is countered by internals, all the most lethal chem grenades rely on reactions like foam and explosions). Chemistry was also hindered because for all the cool stuff you can make, you're still relegated to basically being a vending machine stock boy. The smoke machine gave chemists a far more dynamic option for distributing chems.

**Comment by Iamgoofball** [comment]
Welcome to retarded players 

You're fucked though because joan hates you so RIP your design vision

**Comment by Qbopper** [comment]
itt: robustin tries to set up a strawman argument where I complain it makes things too easy to clean

really makes you think

**Comment by praisenarsie** [comment]
what

**Comment by Qbopper** [comment]
> Qbopper you're sitting here complaining that it makes the station too easy to clean? Really?

this is a strawman argument

you know this is a strawman argument

i won't debate with you because you don't want to and you will not change your mind

please leave me out of this

**Comment by Bawhoppen** [comment]
He also had an autistic shitfest after I moved his precious circuit board to tech storage where circuit boards belong and just wouldn't stop talking about it, until he eventually forced through a PR adding a prebuilt version to chem.

**Comment by Iamgoofball** [comment]
Not all boards should be in tech storage by default and you know it.

**Comment by Bawhoppen** [comment]
Actually they should be? Engineers are supposed to build machines, not chemists.

**Comment by Robustin** [comment]
Because, as we all agreed, tech storage is where backup boards get stored, new machinery only comes from RND, I tried to break that mold but you banished the board to tech storage, so I had to put the completed machine into chemistry for consistency.

Also because the only autistic thing here is you instantly opening a PR moving the board to tech storage before most people even knew it existed because of some deep OCD you have with circuuit boards existing outside of tech storage. 

All I've seen this week is people using the smoke machine for healing, cleaning, and experimenting so it's a hot button for me when I see people still fucking theorycrafting this as a doomsday device or something that utterly displaces other chemist duties.

**Comment by Bawhoppen** [comment]
I don't know why you're saying everyone agrees, because that's a completely baseless claim honestly.
I also don't automatically assume a room called TECH STORAGE (not called TECH BACKUP STORAGE) is absolutely only for backups to be in it, even if it has backups in it. 
Clearly I also wasn't the only one to think this, since multiple maintainers agreed.

**Comment by sawrge** [comment]
all ive seen this week is people using it to spit carpet and colourful reagent everywhere or try to get security to vape whiskey

**Comment by kevinz000** [comment]
itt robustin btfo

**Comment by Robustin** [comment]
itt Kev posts his usual shitpost whenever I argue with someone.

itt kevinz000 btfo

**Comment by evsey9** [comment]
it uses TWO units of each reagent in the machine, that is a very low amount. sure, you can fuck some stuff up with fire or lube - but with fire the machine is destroyed in a minute, with lube - crowbar your way to the machine and press purge contents.

**Comment by Bawhoppen** [comment]
Robustin is playing 4d chess with gamebalance

**Comment by praisenarsie** [comment]
i'll reopen because i was doing something for hippie

**Pull request closed without merging**